apps/test/unit/code-studio/components/libraries/libraryParserTest.js

```diff
@@ -130,10 +175,16 @@ describe('Library parser', () => {
 
       let expectedFunctions = `${firstFunctionName}: ${firstFunctionName},${secondFunctionName}: ${secondFunctionName}`;
```

Comment:
Maybe add a test with a non-empty description

Reply:
Added to: https://codedotorg.atlassian.net/browse/STAR-765